cmd/commands/assets.go

```diff
@@ -233,6 +234,18 @@ func parseFeeRate(ctx *cli.Context) (uint32, error) {
 	return uint32(0), nil
 }
 
+// parseSkipProofCourierPingCheck parses the skip proof courier ping check flag
+// from the CLI context.
+func parseSkipProofCourierPingCheck(ctx *cli.Context) bool {
+	if ctx.IsSet(skipProofCourierPingCheckName) {
```

Comment:
nit: since a boolean flag can only either be set or not, I don't think we need this helper function.
Can just use:
```
	resp, err := client.SendAsset(ctxc, &taprpc.SendAssetRequest{
		TapAddrs:                  addrs,
		FeeRate:                   feeRate,
		SkipProofCourierPingCheck: ctx.Bool(
			skipProofCourierPingCheckName,
		),
	})
```
below.